src/app/storage/storage.router.ts

```diff
@@ -0,0 +1,55 @@
+import { Router } from 'express';
+
+import { getFilesWithKeyword } from '../../utils/getFilesWithKeyword';
+import { fetchStorage, nukeStorage } from './utils';
+
+// Export module for registering router in express app
+export const router: Router = Router();
+const storageRouter = router;
+
+router
+  .route('/storage/:command?')
```

Comment:
sets the base for all `/storage` routes (in case we extend beyond `queries`).
also takes a command, as a poorman's failsafe for dangerous commands (e.g. nuking the storage, etc.)

Reply:
Context from private messages: this is mainly convenience method used in development/testing.

Let's put this endpoint behind a feature flag. We don't want to expose the entire DNAStack storage api, just our query endpoints.

Reply:
Feature flags, Texas-sized 10-4!